jormungandr/src/rest/v0/handlers.rs

```diff
@@ -169,6 +171,9 @@ async fn create_stats(context: &FullContext) -> Result<serde_json::Value, Error>
         "lastBlockContentSize": tip_header.block_content_size(),
         "lastBlockSum": block_input_sum.0,
         "lastBlockFees": block_fee_sum.0,
+        "peerAvailableCnt": nodes_count.all_available_nodes_count().to_string(),
+        "peerUnreachableCnt": nodes_count.all_unreachable_nodes_count().to_string(),
+        "peerQuarantinedCnt": nodes_count.all_quarantined_nodes_count().to_string(),
```

Comment:
I would convert peer -> peers and lose the Cnt suffix.

Reply:
@rinor I tried to follow the convention set forth by the precedent of ```txRecvCnt``` and ```blockRecvCnt```